test/relayserver/RegistrationManager.test.ts

```diff
@@ -312,7 +318,20 @@ contract('RegistrationManager', function (accounts) {
         await revert(id)
       })
 
-      it('send only manager hub balance and workers\' balances to owner (not manager eth balance)', async function () {
+      it('should not send balance immediately after unauthorize (before unstake delay)', async function () {
```

Comment:
Negative tests barely test anything useful. Here you check the old behaviour is not there, but this is not necessary IMO.

Reply:
you're right. its only here because there's a positive test just next to it (I could do multiple asserts in the same test - would it be better? it will make a huge, unreadable single test method.